src/AzureDataFactory.TestingFramework/Models/Pipelines/Pipeline.cs

```diff
@@ -23,13 +23,12 @@ public PipelineActivity GetActivityByName(string activityName)
     }
 
     /// <summary>
-    /// Evaluates the pipeline with the provided parameters. The order of activity execution is simulated based on the dependencies. Any expression part of the activity is evaluated based on the state of the pipeline.
+    /// Validates whether the list of parameters are complete and not duplicate.
     /// </summary>
-    /// <param name="parameters">The global and regular parameters to be used for evaluating expressions.</param>
-    /// <returns></returns>
-    /// <exception cref="PipelineParameterNotProvidedException">Thrown if a required pipeline parameter is not required</exception>
-    /// <exception cref="PipelineDuplicateParameterProvidedException">Thrown if a pipeline parameter is provided more than once</exception>
-    public IEnumerable<PipelineActivity> Evaluate(List<IRunParameter> parameters)
+    /// <param name="parameters"></param>
+    /// <exception cref="PipelineParameterNotProvidedException"></exception>
```

Comment:
The description of the exception was deleted from the previous version - not sure if it was intentional as the name is self-explanatory, but there are other methods that are including the descriptions. We should just keep the consistency on how do we do it.

Reply:
Nice catch! Fixed.